lib/Backends/CPU/CMakeLists.txt

```diff
@@ -30,13 +30,8 @@ target_compile_options(CPURuntime
 # emit LLVM IR for the LTO'ed AOT JIT runtime) and uses the `llvm-link` as the
 # linker to merge the object files.
 set_target_properties(CPURuntime
-                      PROPERTIES
-                        RULE_LAUNCH_COMPILE
-                          "${CLANG_BIN} <DEFINES> <INCLUDES> <FLAGS> -o <OBJECT> -c <SOURCE> # "
-                        RULE_LAUNCH_LINK
-                          "${LLVM_LINK_BIN} -o ${CMAKE_BINARY_DIR}/libjit.bc <OBJECTS> # "
-                        OUTPUT_NAME
-                          libjit.bc)
+                      PROPERTIES LINK_FLAGS "-Wl,-object_path_lto"
```

Comment:
Please dont use `-Wl,` as that prevents some characters in the invocation.  The `-object_path_lto` is specific to Darwin, so this will break non-Darwin builds.  The custom invocation is absolutely required for non-ld64 builds.